metals/src/main/scala/scala/meta/internal/metals/formatting/MultilineString.scala

```diff
@@ -450,7 +450,8 @@ case class MultilineString(userConfig: () => UserConfiguration)
          *```
          */
         case Seq(token: Token.Constant.String, _, _)
-            if isUnfinishedDoubleQuote(token) =>
+            if isUnfinishedDoubleQuote(token) &&
+              (token.pos.endLine == position.getLine - 1) =>
```

Comment:
This is just in case to minimize possible issue. It can now break in a very limited area. Earlier it could break anywhere in the file.